serial.go

```diff
@@ -91,8 +91,13 @@ type Config struct {
 	// CRLFTranslate bool
 }
 
+type ReadWriteFlushCloser interface {
+	io.ReadWriteCloser
+	Flush() error
+}
+
 // OpenPort opens a serial port with the specified configuration
-func OpenPort(c *Config) (io.ReadWriteCloser, error) {
+func OpenPort(c *Config) (ReadWriteFlushCloser, error) {
```

Comment:
That's true. Can you suggest a way to add Flush() so it can be called like `serial.Flush()` while maintaining backward compatibility ?